src/applications/static-pages/translation.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-console */
+import recordEvent from 'platform/monitoring/record-event';
+
+const faqI18Content = {
+  en: {
+    linkTitle: 'Read this page in English',
+    langToggleLink: 'coronavirus-veteran-frequently-asked-questions',
+  },
+  es: {
+    linkTitle: 'Leer esta página en Español',
+    langToggleLink: 'coronavirus-veteran-frequently-asked-questions-esp',
+  },
+};
+
+const configureTranslationLink = (e, currentLang, targetLang) => {
+  const contentDiv = document.getElementById('content');
+  contentDiv.lang = currentLang;
+  e.dataset.lang = currentLang;
+  e.innerText = faqI18Content[targetLang].linkTitle;
+  e.href = faqI18Content[targetLang].langToggleLink;
+  e.onclick = recordEvent({
+    event: 'faq-lang-toggle',
+    targetLang,
+  });
+};
+
+const displayTranslationLink = () => {
+  const i18LinkWrapper = document.getElementById('i18-link-wrapper');
+  if (!i18LinkWrapper) return;
+  const isSpanish =
+    window.location.href.includes('-esp') ||
+    window.location.href.includes('nodeId=14580');
+  const translatableLinks = [
+    // uncomment the below line once we get through staging review and remove references to preview node
+    // 'coronavirus-veteran-frequently-asked-questions',
+    'coronavirus-veteran-frequently-asked-questions-esp',
+    'nodeId=14580',
+  ];
+  const isTranslatable = translatableLinks.some(url =>
+    window.location.href.includes(url),
+  );
+  if (!isTranslatable) return;
```

Comment:
What's the distinction between `isSpanish` and  `isTranslatable `?

Reply:
`isSpanish` is used for setting the html lang attribute, `isTranslatable` is used for whether we need to display the translation link